### PR TITLE
Win 3.4

### PR DESCRIPTION
**Pull request by umangmehta12**
* Linux(Ubuntu, CentOS) 3.2 with data collection API's
* Windows 3.4 with data collection API's

### CODE REVIEW

File: pages/overview/whats_new.md [hunk 1]
```diff
@@ -14,6 +14,16 @@ This page contains the release notes for the different versions of the Affdex SD
 
 ---
 
+***Feb 2017***
+
+##### Windows SDK 3.4
+* Adding the data collection API's. See documentation for [[enableAnalytics()]({{site.baseurl}}/pages/platforms/v3_4/windows/classdocs/affdex-native/classaffdex_1_1_detector.html#af54e0259e824d4425db980546b374497)] and [[disableAnalytics()]({{site.baseurl}}/pages/platforms/v3_4/windows/classdocs/affdex-native/classaffdex_1_1_detector.html#aaac2694f11b4a4308de6f8e3a1127ad7)]
```
Comment (adennie) [inline pages/overview/whats_new.md]: I suggest changing "Adding" to "Added" to match usage on the rest of the page.  Likewise for the Linux blurb below.

**Comment by umangmehta12** [comment]
@adennie yes havent pushed the binaries to developer-portal yet since we are waiting for craig to test them and for @ahamino to accept the story.

File: pages/overview/whats_new.md (hunk 1 above)
Comment (umangmehta12) [inline pages/overview/whats_new.md]: Made the change!

File: pages/overview/whats_new.md [hunk 2]
```diff
@@ -17,10 +17,10 @@ This page contains the release notes for the different versions of the Affdex SD
 ***Feb 2017***
 
 ##### Windows SDK 3.4
-* Adding the data collection API's. See documentation for [[enableAnalytics()]({{site.baseurl}}/pages/platforms/v3_4/windows/classdocs/affdex-native/classaffdex_1_1_detector.html#af54e0259e824d4425db980546b374497)] and [[disableAnalytics()]({{site.baseurl}}/pages/platforms/v3_4/windows/classdocs/affdex-native/classaffdex_1_1_detector.html#aaac2694f11b4a4308de6f8e3a1127ad7)]
```
Comment (adennie) [inline pages/overview/whats_new.md]: oops, rats, sorry I didn't see this the first time...  "API's" should be "APIs" (no apostrophe).

Comment (umangmehta12) [inline pages/overview/whats_new.md]: Fixed

**Pull request merged**